openquake/commonlib/tests/calculators/event_based_test.py

```diff
@@ -130,6 +130,22 @@ def test_case_2(self):
             out['hazard_curve-smltp_b1-gsimltp_b1-ltr_0.csv'])
 
     @attr('qa', 'hazard', 'event_based')
+    def test_case_3(self):  # oversampling
+        out = self.run_calc(case_2.__file__, 'job_2.ini', exports='csv')
+        self.assertEqualFiles(
+            'expected/0-SadighEtAl1997.csv',
+            out['0-SadighEtAl1997.csv'], sorted)
+        self.assertEqualFiles(
+            'expected/1-SadighEtAl1997.csv',
+            out['1-SadighEtAl1997.csv'], sorted)
+        self.assertEqualFiles(
+            'expected/hazard_curve-smltp_b1-gsimltp_b1-ltr_0.csv',
+            out['hazard_curve-smltp_b1-gsimltp_b1-ltr_0.csv'])
+        self.assertEqualFiles(
+            'expected/hazard_curve-smltp_b1-gsimltp_b1-ltr_0.csv',
```

Comment:
No, the files 0.csv and 1.csv should actually be equal and we are testing that. I have added a comment on this respect.